docs/development.md

```diff
@@ -168,7 +168,7 @@ federation controller manager:
 
 ### Cleanup
 
-Follow the [cleanup instructions in the user guide](userguide.md#cleanup).
+Follow the [cleanup instructions in the user guide](README.md#uninstalling-the-chart).
```

Comment:
`README.md` is not under `docs`. So, you'd better use relative links to `README.md`

Reply:
done!